source/intro.tex

```diff
@@ -1177,9 +1177,11 @@
 memory location.
 
 \pnum
-The library defines a number of atomic operations (Clause~\ref{atomics}) and
-operations on mutexes (Clause~\ref{thread}) that are specially identified as
-synchronization operations. These operations play a special role in making
+The library identifies certain operations as
+\defnx{synchronization operations}{synchronization operation}
+(\ref{support.signal}, \ref{util.smartptr.shared.atomic}, \ref{atomics.order},
+\ref{atomics.fences}, Clause~\ref{thread}).
```

Comment:
regarding [support.signal]: "A call to the function signal synchronizes with any resulting invocation of the signal handler so installed."